lib/active_model/serializer/assertions.rb

```diff
@@ -0,0 +1,16 @@
+require 'json-schema'
+
+module ActiveModel
+  class Serializer
+    module Assertions
+      def assert_response_schema(schema_path = nil)
```

Comment:
should be under a test namespace if it's for tests, e.g. `ActiveModelSerializers::Test::Schema`

And would be good for the method not to be dependent on where it's included 

e.g.

```ruby
module Schema
  module_function

  def validate!(schema_full_path, json, strict = true)
    JSON::Validator.validate!(schema_full_path, json, strict: strict) # is this keyword args? we can't support that yet
  end

  def validate_response_schema!(response, schema_path = nil)
    validate!(schema_full_path_for(response.body, schema_path))
  end

  def schema_full_path_for(response, schema_path = nil)
        controller_path = response.request.filtered_parameters[:controller]
        action = response.request.filtered_parameters[:action]
        schema_directory = ActiveModel::Serializer.config.schema_path
        schema_path ||= "#{controller_path}/#{action}.json"
        "#{schema_directory}/#{schema_path}"
  end

```